### ISSUE TITLE
Sleep mode on TV screen displays "No source connected"

### ISSUE THREAD

**Issue by shoten** [evancohen/smart-mirror#505]
I have an old LG TV that I used on my smart mirror (not a PC monitor) it doesn't support CEC.
The script raspi-monitor.sh works perfectly, but unfortunately on my TV when no source are connected it display "No source connected". 
Before the new year version I only used the black layer (and not the script) but since the last version update the "black layer" is not here anymore so I have this text moving around the screen

### Expected Behavior
When I say "Go to sleep" the screen turn black without the need of shuting down the HDMI
Same for "go home" 

### Current Behavior
When I say "Go to sleep" if 
1. I put a command line in the remote config page the hdmi switch off the screen and I have a "no source connected" on my screen
2. I put nothing in the command line the display doesn't change and I have an error in the js inspector 

### Possible Solution
1. Display again a black layer, add an option in the autotimer config "legacyTV" set to "false" by default (or something like that) that I can turn to "true" and show hide a dark layer and not use the script
2. In the script : change the script to add an option "legacy" to show / hide a black image in front of the electon broswer (but I don't know if it's possible, the script should get the pid and kill it after I suppose) or any other elegant way to show a black screen (like activate / deactivate the screensaver)

### Steps to Reproduce (for bugs)

1. configure the commands line in the Autotimer setting to the defaut scripts
2. say "Go to sleep"
3. the HDMI connection is shutdown  
4. the TV start showing 'no source connected"

### Context

The mirror use an old LG TV not compatible CEC (not a PC monitor)

### Your Environment

* Environment name and version (e.g. Raspberry Pi 3, node.js 5.4): 
Rapsberry 3, ps EYE, node v6.9.2 , npm v3.10.9
* Operating System and version (e.g. Raspbian, Ubuntu, OSX): 
Linux raspberrypi 4.4.34-v7+ #930 SMP Wed Nov 23 15:20:41 GMT 2016 armv7l GNU/Linux

**Comment by evancohen** [comment]
@justbill2020 looks like we should bring back the "sleep" shade for people who don't have compatible monitors.

**Comment by justbill2020** [comment]
Yes I'm going to set that up after kids go to bed and have it selectable in config

**Comment by justbill2020** [comment]
It will do either or and still wake with motion or saying the keyword

**Comment by justbill2020** [comment]
Can you assign this to me?
